mono/utils/mono-dl-windows.c

```diff
@@ -72,10 +75,10 @@ mono_dl_open_file (const char *file, int flags)
 		if (!hModule)
 			SetLastError (last_error);
 	} else {
-#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)
+#ifdef HAVE_API_SUPPORT_WIN32_GET_MODULE_HANDLE
```

Comment:
Should be `if` not `ifdef` so that it can be explicitly stated as not present, instead of ambiguous omission. Also should be `GetModuleHandleW`. Plain `GetModuleHandle` is a Win9x compatibility layer.